test/test_helper.exs

```diff
@@ -9,6 +9,7 @@ Mimic.copy(Ecto.Adapters.SQL)
 {:ok, _} = Application.ensure_all_started(:wallaby)
 
 ExUnit.start(capture_log: true)
+Faker.start()
```

Comment:
in normal case we don't need this step, can you share with us why do we need this step @nvminhtue if it is a needed, we can add it into our elixir template :)

Reply:
It was mentioned on official docs: https://github.com/elixirs/faker
Now I understand that the Faker still working without adding to the test_helper.